packages/web/src/exporter.js

```diff
@@ -67,6 +67,7 @@ class EpsagonExporter extends CollectorTraceExporter {
 
       Object.keys(spansList).forEach((spanIndex) => {
         const spanSubList = spansList[spanIndex].spans;
+        diag.debug(`Handle spans for index ${spanIndex}`);
```

Comment:
I love the other method better diag.debug('Handle...', spandIndex) 
I suggest to stick to one method